### PR TITLE
[MIG][13.0] partner_label

### PR DESCRIPTION
**Pull request by ajayjaviya**
Add partner label report

report

### CODE REVIEW

File: partner_label_report/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Adds a label report on partners, to be used for instance in enveloppes or badges.
+
+.. image:: /partner_label_report/static/description/report.png
```
Comment (ivantodorovich) [inline partner_label_report/readme/DESCRIPTION.rst]:
```suggestion
.. image:: ../static/description/report.png
```

File: partner_label_report/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Go to Contacts / Partners
+
+Select the partners you want to print
+
+Click on Print > Label report
+
+.. image:: /partner_label_report/static/description/contacts.png
```
Comment (ivantodorovich) [inline partner_label_report/readme/USAGE.rst]:
```suggestion
.. image:: ../static/description/contacts.png
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
:+1:

**Comment by NL66278** [comment]
Functionality overlaps with this PR for 12.0: https://github.com/OCA/partner-contact/pull/873 However the 12.0 version is more configurable. So I would suggest to migrate that one to 13.0 (add hbrunn's remote to your repo clone, base your branch on his branch and add the changes needed to go from 12.0 to 13.0 if any).

**Comment by ivantodorovich** [comment]
@ajayjaviya it seems tests are failing

**Review by NL66278 — APPROVED** [review]
:+1: Thanks for the migration. LGTM.

**Review by ivantodorovich — APPROVED** [review]
awesome, thanks!

**Comment by ivantodorovich** [comment]
something weird is happening with @OCA-git-bot :sweat_smile: 
should we try again?

**Pull request merged**